sql/moz-fx-data-shared-prod/firefox_desktop_derived/desktop_dau_distribution_id_history_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+friendly_name: Desktop MozillaOnline for Baseline Clients Daily
+description: |-
+  Time series mapping table of distribution_ids from legacy (main) pings for desktop DAU based on the baseline ping
```

Comment:
```suggestion
  Time series mapping of legacy telemetry client's distribution_id to a GLEAN client as historically GLEAN did not collect this information and is required for DAU analysis based off GLEAN telemetry.
```

Reply:
I'm not sure if this is better, but I did find the description a little bit confusing. @irrationalagent WDYT?

Reply:
I think that's fine and accurate